common/OpTestHMC.py

```diff
@@ -272,6 +344,15 @@ def wait_system_state(self, exp_state=OpManagedState.OPERATING, timeout=SYS_WAIT
                 raise OpTestError("Time exceeded for reaching %s" % exp_state)
 
     def is_lpar_in_managed_system(self, mg_system=None, lpar_name=None, remote_hmc=None):
+        '''
+        Check if LPAR is available on a managed system
+
+        :param mg_system: string, managed system name
+        :param lpar_name: string, LPAR name
+        :param remote_hmc: string, remote HMC name
+        :returns: True  - when the LPAR is in managed system
+                  False - when the LPAR is not in managed system
+        '''
         hmc = remote_hmc if remote_hmc else self
         lpar_list = hmc.ssh.run_command(
                    'lssyscfg -r lpar -m %s -F name' % mg_system)
```

Comment:
As Rashi pointed out, remote_hmc is an object and not a strint , containing reference to attributes and fuctions of OpTestHMC()